eng/build.ps1

```diff
@@ -0,0 +1,27 @@
+[CmdletBinding(PositionalBinding=$false)]
+Param(
+  [string] $ArchGroup,
+  [switch] $Release,
+  [switch] $c,
+  [Parameter(ValueFromRemainingArguments=$true)][String[]]$ExtraArgs
+)
+
+$arguments = ""
+
+foreach ($argument in $PSBoundParameters.Keys)
+{
+  switch($argument)
+  {
+    "Debug"      { $arguments += "/p:ConfigurationGroup=Debug " }
+    "Release"    { $arguments += "/p:ConfigurationGroup=Release " }
+    "ExtraArgs"  { $arguments += $ExtraArgs }
+    "c"          { $arguments += "-rebuild " }
```

Comment:
this might be explained on the corefx PR, but just wondering why does "c" represent "-rebuild" ??

Reply:
I have no idea, it might just be a standard-ism: https://github.com/dotnet/standard/blob/master/config.json#L157-L159

Reply:
It is a standard alias only. I created when working in standard as an easy way to do a clean build.